java/cloudbuild-test.yaml

```diff
@@ -23,7 +23,7 @@ steps:
       - '-c'
       - |
         for D in $(find java -type d -depth 1); do
-          cp Dockerfile.ubuntu-bookworm.airlock "${D}"
+          cp Dockerfile.ubuntu-bookworm.airlock $D
```

Comment:
Cloud Build doesn't like the shell variable https://github.com/googleapis/testing-infra-docker/pull/435/checks?check_run_id=35721765297.